offload/test/mapping/declare_mapper_nested_default_mappers_array.cpp

```diff
@@ -50,7 +48,7 @@ int main() {
   sa[1].h = N;
 
   printf("%d %d %d %4.5f %d\n", sa[1].e, sa[1].f.a, sa[1].f.c.a, sa[1].f.b[1],
-         sa[1].f.b == &x[0] ? 1 : 0);
+         sa[1].f.b == &y[0] ? 1 : 0);
```

Comment:
Using `x` instead of `y` in these comparisons was a bug in the test. The fixed test passes with the compiler code change.